ping-payload-extensions/extensions.md

```diff
@@ -0,0 +1,12 @@
+# This document lists defined extensions
+This is a list and short description of all the extensions
+
+- Extensions can be supported by all sub-networks or a subsection
+
+
+| Type number  |  Name | Supported sub-networks  | Short Description  | Is this call Required to Implement  |
+|---|---|---|---|---|
+| [0](extensions/type-0.md)  | Client Info, Radius, and Capabilities | All  | Returns client info e.x. `trin/0.1.1-2b00d730/linux-x86_64/rustc1.81.0`, the nodes radius and a list of enabled extensions  |  Yes  |
+| [1](extensions/type-1.md)  |  Basic Radius Payload | State, Beacon | Provides the nodes Radius  |  Yes  |
+| [2](extensions/type-2.md)  | History Radius Payload  |  History | Provides the nodes radius and ephemeral header count  |  Yes  |
```

Comment:
But these would not be required to implement as they are not part of the standard extensions?

Reply:
If you want to make them standard extensions, well then to remove them it would require a breaking change. If we update to make it so the updating the ping extension the history network uses without breaking changes well then `type 2` can't be a standard extension. It can be required by the history network, but it isn't assumed they have it so they would need to upgrade through `type 0` first.

So it is a trade off. I am not that picky which path we choose

Reply:
If they were a standard extension
- nodes could ping them directly without an `type 0` packet to upgrade
- changing or removing them would require breaking changes

If they were non-standard extensions
- they could still be required by Portal subnetworks, but the peer would need to call `type-0` first and then upgrade to the latest commonly supported ping type by X subnetwork
- this system wouldn't require breaking changes